xlators/features/shard/src/shard.h

```diff
@@ -20,6 +20,8 @@
 #define GF_SHARD_REMOVE_ME_DIR ".remove_me"
 #define SHARD_MIN_BLOCK_SIZE (4 * GF_UNIT_MB)
 #define SHARD_MAX_BLOCK_SIZE (4 * GF_UNIT_TB)
+#define SHARD_MIN_DELETION_INTERVAL 10
```

Comment:
IMO, 10 sec is too low since we don't see someone triggering deletion so frequently and also lead to a negative impact on performance.   I think we can have a default value as SHARD_MIN_DELETION_INTERVAL.

Reply:
Well, this may be useful for testing (as well as tiny, may be down to 4K, shards).